### PR TITLE
[EOY] Add banner to artist pages that are featured in article.

### LINKED ISSUE #539: [EOY] Artist page SEO banner
The existing “this work is in an ongoing auction” banner is rendered server-side and I would also prefer not to have to pull down the full JSON file of featured artists on each client.

- Cache the JSON data per server process.
- If the artist is in the JSON data, render a SEO banner instead of an auction one [here](https://github.com/artsy/force/blob/dacb41a7c51bc2085b97150b811bb17e7e01970c/apps/artist/routes.coffee#L38).
- Remove the ‘edit JSON page’ route so that people can’t easily change the contents and have it go out of date with what’s cached on the server

### PR DESCRIPTION
**Pull request by alloy**
Closes #539

screen shot 2016-12-21 at 14 56 36

### CODE REVIEW

File: apps/artist/components/current_eoy_listing/index.coffee [hunk 1]
```diff
@@ -0,0 +1,18 @@
+_ = require 'underscore'
+fetchEOY2016Lists = require '../../../../components/eoy_artist_list/server.coffee'
+
+module.exports = (artist) ->
+  fetchEOY2016Lists()
```
Comment (craigspaeth) [inline apps/artist/components/current_eoy_listing/index.coffee]: 👍

File: apps/artist/test/components/current_eoy_listing/index.coffee [hunk 2]
```diff
@@ -0,0 +1,21 @@
+_ = require 'underscore'
+Q = require 'bluebird-q'
+data = require '../../../../eoy_2016/fixtures/data.json'
+rewire = require 'rewire'
+currentItem = rewire '../../../components/current_eoy_listing/index'
+currentItem.__set__ 'fetchEOY2016Lists', () -> Q(data)
+
+describe 'current EOY listing', ->
```
Comment (craigspaeth) [inline apps/artist/test/components/current_eoy_listing/index.coffee]: 👌

**Comment by craigspaeth** [comment]
Solid. Thanks!

**Comment by alloy** [comment]
Yay!

**Pull request merged**